app/models/goal.py

```diff
@@ -1,5 +1,13 @@
 from app import db
 
-
 class Goal(db.Model):
-    goal_id = db.Column(db.Integer, primary_key=True)
+    goal_id = db.Column(db.Integer, primary_key=True, autoincrement=True)
```

Comment:
My personal preference is to name id attributes just `id` and then when you have a goal object you can write `goal.id` instead of `goal.goal_id`.